### PR TITLE
Add option to sync from beginning of log file #83

### PR DESCRIPTION
**Pull request by azell**
https://github.com/papertrail/remote_syslog2/issues/83

New command line and YAML parameter to specify whether the tail should begin at the end (default) or beginning of each file.  I would rather uniq my log statements than miss out on them entirely as can be the case during log rotation and remote_syslog's 10 second check for new files.

### CODE REVIEW

File: config_manager.go [hunk 1]
```diff
@@ -53,6 +54,7 @@ type ConfigManager struct {
 		Severity        string
 		Facility        string
 		Poll            bool
+		Rewind          bool
```
Comment (troy) [inline config_manager.go]: Good choice of option name :)

**Comment by troy** [comment]
@azell: Thanks! Heads up that merging this may be delayed due to a fairly large refactor that couldn't easily be handled incrementally. It's winding through a review/test cycle now and may prevent merging this first.

@leonsodhi: Giving this to review as you see fit.

**Comment by azell** [comment]
Any update on this request?  Thanks!

**Comment by troy** [comment]
@azell No news. It's on our radar but in line behind some required improvements. While I can't say whether it'll be a week or a month, it won't get dropped.

**Pull request closed without merging**